code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/LahtachevYaroslavTest.java

```diff
@@ -0,0 +1,139 @@
+package ru.unn.agile.ComplexNumber.Model;
+
+import org.junit.Test;
+import ru.unn.agile.ComplexNumber.model.ComplexNumber;
+
+import static org.junit.Assert.*;
+
+public class LahtachevYaroslavTest {
+
+    private final double delta = 0.001;
+
+    @Test
+    public void defaultConstructorMakeZeroNumber() {
+        ComplexNumber complexNumber = new ComplexNumber();
+        assertEquals(new ComplexNumber(0, 0), complexNumber);
+    }
+
+    @Test
+    public void constructorStringAndDoubleReturnEqualNumber() {
+        ComplexNumber complexNumber1 = new ComplexNumber("1", "1");
```

Comment:
strange name

Reply:
may be like "areNumbersWithDifferentInitialTypesEqual"?

Reply:
yeah, it's nice:)